src/main/java/seedu/address/model/Model.java

```diff
@@ -8,11 +8,12 @@
 import seedu.address.model.appointment.Appointment;
 import seedu.address.model.appointment.AppointmentDateTime;
 import seedu.address.model.person.Person;
+import seedu.address.model.schedule.ScheduleModel;
 
 /**
  * The API of the Model component.
  */
-public interface Model {
+public interface Model extends ScheduleModel {
```

Comment:
I am slightly concerned about having Model extend ScheduleModel in terms of design, but I understand why it's done this way for abstraction and to maintain compatibility. If there is no better solution then it LGTM. Perhaps in a future PR we can abstract the different model interfaces out (for tutor, appointment, etc) and then have Model extend all of them for consistency.

Reply:
Noted, then I'll shift it back to Model for meantime. If we decided to adopt this approach then I'll reinstate back in this file